backend/create_account_table.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE Accounts(
-	id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+	account_id INT UNSIGNED NOT NULL AUTO_INCREMENT,
 	name VARCHAR(256) NOT NULL UNIQUE,
 	password VARCHAR(32) NOT NULL,
-	PRIMARY KEY(id)
+	PRIMARY KEY(account_id)
 );
```

Comment:
Changing the column name has pretty major repercussions and will make merging difficult. Please revert the changes made to this file and related changes